ciphers/autoclave.py

```diff
@@ -0,0 +1,111 @@
+def encrypt(plaintext, key) -> str:
```

Comment:
```suggestion
def encrypt(plaintext, key) -> str:
```
These parameters need to be typehinted